R/run-url.R

```diff
@@ -25,7 +26,7 @@
 #' runUrl("https://github.com/rstudio/shiny_example/archive/master.zip",
 #'  subdir = "inst/shinyapp/")
 #' }
-runUrl <- function(url, filetype = NULL, subdir = NULL, ...) {
+runUrl <- function(url, filetype = NULL, subdir = NULL, destDir = NULL, ...) {
```

Comment:
For consistency with the other arguments, I'd please use `destdir` instead of `destDir`.